capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java

```diff
@@ -371,10 +372,7 @@ private void handleOnBackPressed() {
         activity.getOnBackPressedDispatcher().addCallback(mFragment.getViewLifecycleOwner(), new OnBackPressedCallback(true) {
             @Override
             public void handleOnBackPressed() {
-                trackCameraScreenEvent(CameraScreenEvent.EXIT);
-                setEnabled(false);
-                remove();
-                mFragment.getActivity().getOnBackPressedDispatcher().onBackPressed();
+                onBackPressed();
```

Comment:
[Request] Here we should only track the exit event because the navigation is handled by the fragment manager / navigation component:
```suggestion
                trackCameraScreenEvent(CameraScreenEvent.EXIT);
                setEnabled(false);
                remove();
```

It might have unexpected side-effects if we pop the back stack and then under the hood the fragment manager or the navigation component pops it again.

Reply:
In this case, the `CameraFragmentImpl` overrides `handleOnBackPressed` behavior by adding its own callback. `remove` only removes the callback, but does not execute the back press event. Because of this,
not calling our own `onBackPressed` method does not exit the camera flow. The user would need to tap a second time on the back button in order to exit it (because then the custom back callback would have been removed).